README.md

```diff
@@ -81,10 +99,22 @@ These are the current planned major milestones:
 
 4. [DONE] Optimize some important performance bottlenecks.
 
-5. Generate useful errors for code that uses unsupported Python
+5. [PARTIALLY DONE] Generate useful errors for code that uses unsupported Python
    features instead of crashing or generating bad code.
 
-6. Release a version of mypy that includes a compiled mypy.
+6. [DONE] Release a version of mypy that includes a compiled mypy.
+
+7a. More feature/compatability work. (100% compatability is distinctly
+    an anti-goal, but more than we have now is a good idea.)
+
+7b. Support compiling some other prominent Python tool.
+    (Black, maybe? Let us know if you maintain a Python tool
```

Comment:
I maintain Black and this seems like a good idea :) Off the top of my head, the main difficulty is probably going to be dataclasses, which Black uses heavily.

Reply:
Yeah, I looked at Black a few months ago and came to that conclusion too and opened https://github.com/mypyc/mypyc/issues/495. (After I implemented yield from basically because of Black). There is some use of async/await but that can probably be hacked around or refactored or something.